File: output/gemma4-e4b_IO_1/avatar/Python/Java/codeforces_131_B.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class codeforces_131_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Read N (required to consume the first line)
        if (!scanner.hasNextInt()) {
            System.out.println(0);
            return;
        }
        int n = scanner.nextInt();

        // Map to store counts: value -> count
        Map<Integer, Long> counts = new HashMap<>();
        
        // Read all remaining tokens as integers
        while (scanner.hasNextInt()) {
            int num = scanner.nextInt();
            counts.put(num, counts.getOrDefault(num, 0L) + 1);
        }
        
        scanner.

File: output/gemma4-e2b_IO_1/avatar/Java/Python/codeforces_131_B.py
import sys

def solve():
    # Read all input tokens
    data = sys.stdin.read().split()
    
    if not data:
        return

    try:
        # The first token is n
        n = int(data[0])
        
        # The remaining tokens are the counts
        if len(data) < n + 1:
            # Not enough counts provided
            return
            
        counts = [int(x) for x in data[1:n+1]]
    except ValueError:
        # Handle case where input might not be integers
        return

    # Initialize cnt array of size 21 to zeros
    cnt = [0] * 21

    # Process the counts
    for i in range(n):
        # The Java code uses scan.nextInt() sequentially.
        # The i-th count read corresponds to counts[i]
        # Index calculation: counts[i] + 10
        index = counts[i] + 10
        
        # Ensure index is within bounds [0, 20] before incrementing
        if 0 <= index <= 20:
            cnt[index] += 1

    # Calculate result
    res = 0
    
    # Calculate sum of products: cnt[i
